packages/terra-infinite-list/tests/wdio/infinite-list-spec.js

```diff
@@ -0,0 +1,54 @@
+/* global browser, Terra */
+
+const viewports = Terra.viewports('small');
+
+describe('InfiniteList', () => {
+  describe('Displays an infinite list with loading indicator', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/infinite-list/infinite-list-loading');
+      browser.waitForVisible('#test-infinite-list');
+    });
+
+    Terra.should.matchScreenshot('#test-infinite-list', { selector: '#test-infinite-list', viewports });
+  });
+
+  describe('Displays an infinite list with updating indicator', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/infinite-list/infinite-list-updating');
+      browser.waitForVisible('#test-infinite-list');
+    });
+
+    Terra.should.matchScreenshot('#test-infinite-list', { selector: '#test-infinite-list', viewports });
+  });
+
+  describe('Displays an infinite list with selection', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/infinite-list/infinite-list-selection');
+      browser.waitForVisible('#test-infinite-list');
+    });
+
+    Terra.should.matchScreenshot('#test-infinite-list', { selector: '#test-infinite-list', viewports });
+  });
+
+  describe('Displays an infinite list with virtual dom from top', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/infinite-list/infinite-list-virtual');
+      browser.waitForVisible('#test-infinite-list');
+      browser.pause(150);
```

Comment:
What is the pause needed for? Didn't we already wait for the visible element up above?

Reply:
We are waiting for additional javascript logic to execute and potentially trigger another render, so we need to ensure this has time before taking a screenshot.